airflow/www/templates/airflow/dag.html

```diff
@@ -82,6 +82,9 @@
   {% endif %}
   {% if external_log_name is defined %}
   <meta name="external_log_name" content="{{ external_log_name }}">
+  {% if appbuilder.sm.can_edit_dag(dag.dag_id) %}
+  <meta name="can_edit" content="{{ appbuilder.sm.can_edit_dag(dag.dag_id) }}">
```

Comment:
https://github.com/apache/airflow/blob/505af06303d8160c71f6a7abe4792746f640083d/airflow/www/views.py#L607-L626

Reply:
Oh, that's not the right thing (but still, use `dag.can_edit` instead.)

Reply:
Should I use that instead of the current meta `content`?

Also, what I meant is that a lot of these endpoints are checking multiple permissions, not just dag.can_edit.

(ie: [/clear](https://github.com/apache/airflow/blob/main/airflow/www/views.py#L2014)
```
@expose('/clear', methods=['POST'])
@auth.has_access(
    [
        (permissions.ACTION_CAN_EDIT, permissions.RESOURCE_DAG),
        (permissions.ACTION_CAN_DELETE, permissions.RESOURCE_TASK_INSTANCE),
    ]
)

Reply:
_More gracfully_, yes. The error response is still an html redirect. But the UI won't crash.
I feel like fixing the error response from the webserver should be in another PR?